README.md

```diff
@@ -156,6 +157,9 @@ const token = await reply.generateCsrf()
 
 You can also pass the [cookie serialization](https://github.com/fastify/fastify-cookie) options to the function.
 
+The option `userInfo` is required if `getUserInfo` has been specified in the module option.
```

Comment:
We should add more description about the `userInfo` and how it works inside `@fastify/csrf`. When I first glance through this PR, I think it will expose the credential info. But I know I was wrong after looking through `@fastify/csrf`.
So, it is worth notice that the `userInfo` will be hashed inside the `csrf token` and will not be exposed directly to public.

Reply:
added, PTAL